projects/core/src/cart/utils/utils.ts

```diff
@@ -1,9 +1,13 @@
 import { Cart } from '../../model/cart.model';
 import { OCC_USER_ID_ANONYMOUS } from '../../occ/utils/occ-constants';
 
-export function getCartIdByUserId(cart: Cart, userId: string) {
+export function getCartIdByUserId(cart: Cart, userId: string): string {
   if (userId === OCC_USER_ID_ANONYMOUS) {
     return cart.guid;
   }
   return cart.code;
 }
+
+export function isFreshCartId(cartId: string): boolean {
```

Comment:
What do you think about changing nomenclature from `fresh` to `temporary`?

Reply:
Yep, suits better